src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/ProPlanControls.spec.jsx

```diff
@@ -325,21 +344,138 @@ describe('ProPlanControls', () => {
 
       it('renders the proceed to checkout for the update button', async () => {
         setup({ planValue: Plans.USERS_BASIC })
-        render(<ProPlanControls />, { wrapper: wrapper() })
+        render(<ProPlanControls {...props} />, { wrapper: wrapper() })
 
         const proceedToCheckoutButton = await screen.findByRole('button', {
           name: /Proceed to Checkout/,
         })
         expect(proceedToCheckoutButton).toBeInTheDocument()
       })
 
+      xdescribe('when the user has team plans available', () => {
```

Comment:
Most of these tests don't apply cause I had intertwined the team logic before but had written the tests, so I skipped them rather than erasing them. I can exclusively do these in a subsequent PR if you think that makes sense